main/appserver/tests/rq.sh

```diff
@@ -199,12 +200,21 @@ OPTIND=1
 output_file=""
 verbose=0
 
-while getopts ":b:t:g:a" opt; do
+while getopts ":b:t:g:a:l" opt; do
     case "$opt" in
     b)	branch=$OPTARG;;
     t)  test_ids=($OPTARG);;
     a)  test_ids=(`list_test_ids`);;
 	g)  test_ids=(`list_group_test_ids $OPTARG`);;
+	l)	test_ids=(`list_test_ids`)
+		printf "###################################################################\n"
```

Comment:
Extra decorations will make parsing of the output difficult, just a simple echo (line 213) should be enough.